### PR TITLE
Disable Radius for non-radii layers

### PR DESCRIPTION
**Pull request by volfied**
Addresses #3452 

I've also noticed that we've kept `Vector.jsx` around without requiring it, so cleaned it up.

### CODE REVIEW

File: src/js/jsx/sections/style/Radius.jsx [hunk 1]
```diff
@@ -77,7 +77,10 @@ define(function (require, exports, module) {
          * @param {object} props
          */
         _calculateState: function (props) {
-            var layers = props.document.layers.selected.filter(function (layer) {
+            var disabled = props.disabled || props.document.layers.selected.some(function (layer) {
```
Comment (iwehrman) [inline src/js/jsx/sections/style/Radius.jsx]: I think you've got the wrong quantifier here. You want "all selected layers do not have radii" but this is "some selected layer does not have a radii." As long as the control is valid for some layer it should be enabled, and the invalid layers should be filtered out in the handler.

File: src/js/jsx/sections/style/Vector.jsx [hunk 2]
```diff
@@ -1,65 +0,0 @@
...
-                    <Radius
-                        document={this.props.document}
-                        layers={vectorLayers} />
-                </div>
-            );
-        }
-    });
-
-    module.exports = Vector;
-});
```
Comment (iwehrman) [inline src/js/jsx/sections/style/Vector.jsx]: :+1:

**Comment by volfied** [comment]
@iwehrman I've made the changes. I've also noticed that we had to account for all layers in the selection to make sure we would re-render correctly.

However, as it stands, mixed appearances panel will always disable Radius when layer kinds are mixed, I will open a new issue for this as I feel like the entire Mixed Appearance panel requires work on this.

**Comment by iwehrman** [comment]
LGTM.

**Pull request merged**